app/build.gradle

```diff
@@ -89,4 +107,23 @@ android {
     lintOptions {
         abortOnError false
     }
+}
+
+// about.html is templatized so that we can automatically insert
+// our version string in to it at build time.
+task filterAbout {
+    inputs.files files(['src/main/templates/about.html',
+                        'src/main/AndroidManifest.xml'])
+    outputs.file 'src/main/assests/about.html'
+} << {
```

Comment:
Oooh, thanks to @mfietz for pointing out it was 'assests' -> 'assets'.

Perhaps the outputs.file isn't really needed.